book/docs/classes/OpusPool/Methods/getHealthFactorForUser.md

```diff
@@ -0,0 +1,22 @@
+### getHealthFactorForUser
+
+▸ **getHealthFactorForUser**(`mintedAssets`, `stakedAssets`): `Promise`\<[`OsTokenPositionHealth`](../../../enums/OsTokenPositionHealth.md)\>
+
+Retrieves the vault position health for the user
```

Comment:
Maybe we can add some minimal info about what health means and that health under a certain value will not allow minting.